php/src/Pet/PetDTO.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Pet;
+
+use App\Core\AbstractDTO;
+use Cassandra\Uuid;
+
+class PetDTO extends AbstractDTO
+{
+
+    public $ownerId;
+    public $chipId;
+    public $color;
+    public $breed;
+    public $species;
+    public $gender;
+    public $age;
+    public $weight;
+    public $address;
+    public $name;

```

Comment:
Would be nice to add a DocBlock for each of these.

Reply:
What about use type declaration?

Reply:
I don' t think we can use type declaration in the PHP version used in this project